src/cls/IPM/Utils/Module.cls

```diff
@@ -2288,19 +2290,47 @@ ClassMethod %EvaluateSystemExpression(pString As %String) As %String [ Internal
 
 	Set dbRole = ..GetDatabaseRole()
 	Set result = ..%RegExReplace(result, "dbrole", dbRole)
-
-	Set result = ..%RegExReplace(result, "authK5API", $$$AutheK5API)
-	Set result = ..%RegExReplace(result, "authPassword", $$$AutheCache)
-	Set result = ..%RegExReplace(result, "authUnauthenticated", $$$AutheUnauthenticated)
-	Set result = ..%RegExReplace(result, "authLDAP", $$$AutheLDAP)
-	Set result = ..%RegExReplace(result, "authDelegated", $$$AutheDelegated)
-	Set result = ..%RegExReplace(result, "authLoginToken", $$$AutheLoginToken)
-	Set result = ..%RegExReplace(result, "authTwoFactorSMS", $$$AutheTwoFactorSMS)
-	Set result = ..%RegExReplace(result, "authTwoFactorPW", $$$AutheTwoFactorPW)
+	Set result = ..%RegExReplace(result, "globalsDbRole", dbRole)
 
 	Return result
 }
 
+ClassMethod %EvaluateMacro(pString As %String) As %String [ Internal ]
+{
+	Set tString = pString
```

Comment:
For better performance, only do this if the string contains `$$$`. If it doesn't, just return pString as is